### PR TITLE
*: remove useless code.

### PR DESCRIPTION
**Pull request by hanfei1991**
remove some useless code when resolving index.
@shenli @coocood @zimulala PTAL

### CODE REVIEW

File: plan/logical_plan_builder.go [hunk 1]
```diff
@@ -1120,6 +1120,7 @@ func (b *planBuilder) buildUpdateLists(list []*ast.Assignment, p LogicalPlan) ([
 		offset := schema.GetColumnIndex(col)
 		if offset == -1 {
 			b.err = errors.Trace(errors.Errorf("could not find column %s.%s", col.TblName, col.ColName))
+			return nil, nil
```
Comment (tiancaiamao) [inline plan/logical_plan_builder.go]: Does the old code contains bug? if any bug involved, please add test case.

Comment (hanfei1991) [inline plan/logical_plan_builder.go]: No, It will never be -1

**Comment by coocood** [comment]
LGTM

**Comment by hanfei1991** [comment]
@tiancaiamao PTAL

**Comment by tiancaiamao** [comment]
LGTM

**Pull request merged**